docs/modules/release-notes/pages/5-5-0.adoc

```diff
@@ -12,18 +17,17 @@
 For more details on new features, see xref:ROOT:whats-new.adoc[What's new in 5.5].
 
 == Breaking Changes
-There are no breaking changes in this release.
+* For {open-source-product-name} users only - CP Subsystem is now an {Enterprise Edition}-only feature
 
 == Enhancements
-* Replace smart-routing with cluster-routing options (#2489)
+* Replace smart-routing with cluster-routing options
 * Add Debezium 2.x MySQL and PostgreSQL APIs 
 * Introduce queue cap props for persistence
 * Allow EE member to join OS cluster 
 * Client connectivity logging 
 * Add needsFsync to Clear task 
 * Minimum cluster version checks for subset routing 
 * Fail fast on no memberGroups returned with SUBSET enabled 
```

Comment:
```suggestion
```
This is not needed for release notes ("subset routing", now `MULTI_MEMBER` routing, is first released in 5.5)